src/Helpers/index.js

```diff
@@ -32,9 +32,9 @@ const CIRCLE_POINTS_CONFIG = 64;
 export const UNITS = ['kilometers', 'meters', 'miles', 'feet'];
 
 /**
- * Convert the radius into the unit given
- * @param {Number} radius unit
- * @param {String} unit to convert to if necessary
+ * Convert the radius on given unit to a compatible unit for turf/circle
+ * @param {Number} radius on given unit
+ * @param {String} unit of radius given
```

Comment:
(Nitpicking)
I would suggest something closer to this :
```
* Convert the given radius into the given unit and returns a turf/circle compatible object.
* @param {Number} radius of given unit
* @param {String} unit of given radius
```

Reply:
Not exactly, the radius is not *converted* into the given unit, the radius is *assumed* to be in the given unit and converted into compatible unit for turf/cicle